docs/source/tools/mythxvsc/index.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+.. _tools.mythxvsc:
+
+MythXvsc
+============================
+
+MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+The extensions provides:
+
+- Smart contract compilation (via solidity VSCode extension)
+- Log in to MythX platform
+- AST extraction from compiled source
+- Submission of analysis
+- Displaying analysis result in VSCode in a linting fashion
+
+
+
+
+Dependencies
+-----
+
+The MythXvsc extension depends on Juan Blanco’s solidity extension_. Make sure to download this from Visual Studio Code Marketplace before installing MythXvsc.
+
+.. _extension: https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity
+
+Installation
+-----
+
+Install from the Visual Studio marketplace here_.
```

Comment:
Stylistic: Instead of 

> Do a thing _here_:

Suggest:

> _Do a thing_

See: https://medium.com/@heyoka/dont-use-click-here-f32f445d1021